### ISSUE TITLE
Release WorFBench training data on Hugging Face

### ISSUE THREAD

**Issue by NielsRogge** [zjunlp/WorFBench#1]
Hi @Ningyu 🤗

I'm Niels and work as part of the open-source team at Hugging Face. I discovered your work through Hugging Face's daily papers as yours got featured: https://huggingface.co/papers/2410.07869.
The paper page lets people discuss about your paper and lets them find artifacts about it (your dataset for instance),
you can also claim the paper as yours which will show up on your public profile at HF.

It's great to see the test set for WorFBench is already released on Hugging Face!
Would you also like to host the training data on the Hub? This would make it easier for researchers to train and evaluate their models on your benchmark.

Let me know if you're interested/need any guidance.

Kind regards,

Niels
ML Engineer @ HF 🤗

**Comment by GoooDte** [comment]
Hi Niels,

Thank you for your attention. When we uploaded the test data, we also uploaded the training data simultaneously. You can find it at the following address: https://huggingface.co/datasets/zjunlp/WorFBench_train

**Comment by NielsRogge** [comment]
Awesome!

I've opened a PR to improve the dataset card: https://huggingface.co/datasets/zjunlp/WorFBench_train/discussions/1

**Comment by GoooDte** [comment]
Thank you!
